tests/test_absorption_spectrum.py

```diff
@@ -46,7 +46,7 @@
 def test_voigt_profiles():
```

Comment:
Seems like this test was meant to compare the two functions. We probably don't need to keep it.